src/http/get-create_part_v2/index.js

```diff
@@ -49,9 +51,11 @@ exports.handler = async (req) => {
     ret[filename] = [roots.length - 1, root.toString('base32')]
   }
 
+  // create a root pointing to the roots of each file
```

Comment:
we should maybe call out that the root is a list. this has been a little confusing for people for familiar with filecoin because the default workflow in the lotus command is that you pass it one “real” file and it turns it into a unixfs file and makes the root of the CAR file the root of the unixfs file.